types/mempool/mempool_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	protov2 "google.golang.org/protobuf/proto"
 
 	"cosmossdk.io/log"
+	"cosmossdk.io/x/gov"
```

Comment:
We need to remove this dep

Reply:
this is needed for the test, dependency removal im proposing we do after the spinout. Ill focus on getting this all done this week

Reply:
I understand, but I think this test does not require it.

```difff
func unmarshalTx(txBytes []byte) (sdk.Tx, error) {
-	cfg := moduletestutil.MakeTestEncodingConfig(distribution.AppModuleBasic{}, gov.AppModuleBasic{})
+	cfg := moduletestutil.MakeTestEncodingConfig(distribution.AppModuleBasic{})
	return cfg.TxConfig.TxJSONDecoder()(txBytes)
}
```

This still works and the dep is then removed from the SDK.